### ISSUE TITLE
Fails to copy many files as root to ext4-formatted external drive

### ISSUE THREAD

**Issue by tsujan** [lxqt/pcmanfm-qt#646]
This happens almost always:

When I try to copy the contents of `/var/cache/pacman/pkg` (previously, the similar folder in Debian), that contains thousands of files, to an ext4-formatted external drive with `pcmanfm-qt` (to updated another system without downloading gigabytes of packages), either the process is stopped without showing any info or there are many corrupted files. This doesn't have any specific relation to `/var/cache/pacman/pkg`; that folder just serves as an example.

When I do the same thing with `spacefm` -- or before the silly changes in KDE's root access, with `dolphin` -- or just in a terminal, the files will be copied without problem.

I haven't tested with NTFS-formatted drives and have never copied thousands of files to a USB flash or from one partition to another but I think it should also happen when copying thousands of files from a partition to another -- just a guess,.

The problem should be in `libfm` because it also existed in the old versions of `pcmanfm-qt`.

**Comment by tsujan** [comment]
I add that copying hundreds of 100-200-Mib files to a USB flash takes a lot of time to finish. Each file is copied fast but there are long intervals between copyings.

This isn't a pcmanfm-qt issue because it happens with all file managers and with all machines or USB sticks I've tried. It's related to kernel's "Transparent Hugepages", a problem that has existed since that feature was added to the Linux kernel. However, ii'l be good if a workaround is found in file management (there is this manual workaround: `sudo bash -c 'sudo bash -c 'echo "never" > /sys/kernel/mm/transparent_hugepage/enabled'` and echoing "always" after the job is finished).

Anyhow, the main issue of this report is more serious.
